spec/lib/action_kit/helper_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe ActionKit::Helper do
+  before do
+    ENV['AK_PASSWORD'] = 'ak_password'
+    ENV['AK_USERNAME'] = 'ak_username'
```

Comment:
do we want to be messing with ENV vars in tests? we had some weird inconsistencies between environments recently that I think were cause by this, no?